manifests/charts/istio-control/istio-discovery/files/waypoint.yaml

```
@@ -173,13 +173,12 @@ spec:
           successThreshold: 1
           timeoutSeconds: 1
         securityContext:
-          privileged: true
+          privileged: false
           runAsGroup: 1337
           runAsUser: 0
           capabilities:
-            add:
-            - NET_ADMIN
-            - NET_RAW
+            drop:
+            - all
```

```suggestion
            - ALL
```

not sure it matters but usually caps is used

---

Yeah makes sense, not sure where I picked that from.